packages/eas-cli/src/credentials/ios/appstore/experimental.ts

```diff
@@ -0,0 +1,26 @@
+import { BundleId, Profile } from '@expo/apple-utils';
+import { boolish } from 'getenv';
+
+export const USE_APPLE_UTILS = boolish('USE_APPLE_UTILS', false);
+
+export async function getProfilesForBundleIdAsync(bundleIdentifier: string): Promise<Profile[]> {
+  const [bundleId] = await BundleId.getAsync({
+    query: {
+      filter: {
+        identifier: bundleIdentifier,
+      },
+    },
+  });
+  if (bundleId) {
+    return bundleId.getProfilesAsync();
+  }
+  return [];
+}
+
+export async function getBundleIdForIdentifierAsync(bundleIdentifier: string): Promise<BundleId> {
+  const bundleId = await BundleId.findAsync({ identifier: bundleIdentifier });
```

Comment:
what's the difference between `BundleId.findAsync` and `BundleId.getAsync`?

Reply:
getAsync accepts a complex query and returns an array of bundle ids, it also has potential to page results. findAsync is a convenient wrapper for retrieving just the item matching a bundle id.